source/dbo/Stored Procedures/Monitoring/uspUpdateTableCounts.sql

```diff
@@ -0,0 +1,77 @@
+﻿CREATE PROCEDURE [dbo].[uspUpdateTableCounts]
+AS
+	INSERT INTO [TableCounts] (CountTime
```

Comment:
Should this check there isn't already a row for today's date? I'm not sure this would add very much value.

Reply:
Yeah I suspect this will add more complications than it is worth. Also, we might want to run the job after a release and check the counts haven't gone down, even if the job ran earlier that morning. I think I'll pass on this one, unless anyone objects?

Reply:
Agreed, I don't think we want to limit how often the job can run